lib/ai_helper/assistant.rb

```diff
@@ -164,7 +164,7 @@ def generate_image_caption(upload, user)
             feature_name: "image_caption",
           )
 
-        raw_caption.delete("|").squish
+        raw_caption.delete("|").squish.truncate_words(50)
```

Comment:
Maybe pull this into a constant? 

Also, does this include ellipsis automatically?